sqlstore/src/main/resources/db/migration_beforeMigrate/05_add_library.sql

```diff
@@ -0,0 +1,64 @@
+-- StartNoTest
```

Comment:
Many of the add X files actually add a type X, but it seems like these and the ones in the `add_box` file actually add Ys and Zs. Maybe rename them to `add_library_adjacent` or `add_related_to_library`?

Reply:
Okay. `add_library_parameters`. I didn't necessarily intend a 1:1 correspondence between procedures and files.